<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>1100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>波形记录</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>360</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="contore">
    <item>
     <widget class="QPushButton" name="open_port">
      <property name="text">
       <string>打开串口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="close_port">
      <property name="text">
       <string>关闭串口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="save_data">
      <property name="text">
       <string>停止接收保存数据</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="datasend">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>820</y>
     <width>561</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>发送控制</string>
   </property>
   <widget class="QTextEdit" name="send_text_window">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="send_button">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>发送数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button_1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>清空数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send_modl">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>810</y>
     <width>561</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>系统输出状态</string>
   </property>
   <widget class="QTextBrowser" name="Receive_text_window">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清空接收数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="receive_modl">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>10</y>
     <width>261</width>
     <height>335</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="port">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>端口号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="com"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="botlive">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>波特率</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="baud">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string>9600</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="datanum">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>数据位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="bit">
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>停止位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="stopbit">
        <property name="currentText">
         <string>1</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="maxVisibleItems">
         <number>10</number>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>校验位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="jiaoyan">
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="dispychose">
      <property name="title">
       <string>数据通道使能</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="channel1">
         <property name="text">
          <string>通道一</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="channel2">
         <property name="text">
          <string>通道二</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="channel3">
         <property name="text">
          <string>通道三</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="channel4">
         <property name="text">
          <string>通道四</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="channel5">
         <property name="text">
          <string>通道五</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QChartView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>911</width>
     <height>771</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
